test_conformance/common/vulkan_wrapper/opencl_vulkan_wrapper.cpp

```diff
@@ -550,7 +550,7 @@ clExternalMemory::clExternalMemory(const clExternalMemory &externalMemory)
 clExternalMemory::clExternalMemory(
     const VulkanDeviceMemory *deviceMemory,
     VulkanExternalMemoryHandleType externalMemoryHandleType, uint64_t size,
-    cl_context context, cl_device_id deviceId)
+    cl_context context, cl_device_id deviceId, cl_bool useNameForImport)
```

Comment:
Why not add new enum?

Reply:
Yeah, looks like when I was writing this there was no CL_*_HANDLE_OPAQUE_WIN32_NAME_KHR, but since it is there now, will remove the added param and add them in VulkanExternalMemoryHandleType and VulkanExternalSemaphoreHandleType to use those instead.